torch/csrc/jit/mobile/tensor_hash.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
Same comment as before here. It seems tensor_value_has is used only in export_module.cpp. It can be declared and used there without adding this separate file.

Reply:
Good call! It was used in both export_module.cpp and import.cpp in my previous commit, and with the latest implementation it's only used in export_module.cpp. I will move the to export_module.cpp and remove this file.

Reply:
Resolve in the latest commit.